care/users/api/serializers/lsg.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
```

Comment:
Two points:
- Is `lsg` a good name? We plan to make this usable across India right? Is lsg the name used across? I checked "Karantaka lsg" and "Tamil Nadu lsg" and couldn't find them using this name. How about `address`?
- I suggest putting these in a separate app altogether since these are not really exclusive to a user but used by facility too.

Reply:
LSG (Local Self Governing Body) is the actual word used in the rule books. But yeah, we dont use it colloquially. 

Moving to different app? I retained it there because of District Choices, I don't see LSG growing out of this heirarchy, also sinces Users are at LSG level, we can retain there.

Reply:
Ok

Reply:
@vigneshhari @biswaz thoughts ?

Reply:
lsg is the standard name for storing local governing bodies